pkg/cluster/builder_test.go

```diff
@@ -74,7 +84,7 @@ func TestNewSpecWithBundlesOverrideValid(t *testing.T) {
 	v := version.Info{GitVersion: "v0.0.1"}
 	reader := files.NewReader()
 	b := cluster.NewFileSpecBuilder(reader, v,
-		cluster.WithReleasesManifest("testdata/invalid_release_version.yaml"),
+		// cluster.WithReleasesManifest("testdata/invalid_release_version.yaml"),
```

Comment:
typo??

Reply:
"invalid_release_version.yaml" does not exist so the test always fails
I've replaced it with "simple_release.yaml"